Common/ac/gamesetupstruct.h

```diff
@@ -100,7 +100,7 @@ struct GameSetupStruct: public GameSetupStructBase {
 
 
     GameSetupStruct();
-    ~GameSetupStruct();
+    ~GameSetupStruct() override;
```

Comment:
Ha! This actually points to mistake. I think GameSetupStructBase's destructor should not be virtual, because there this inheritance was never supposed to be virtual, its just one struct extending another.

Reply:
I also noticed `UnInit` was not actually overriding in the graphics drivers, but marked as virtual anyway.

Are these things you'd like fixed in this branch?